hack/overlays/set-envoy-daemonset-resources-limits.yaml

```
@@ -0,0 +1,28 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match by=overlay.all,expects="1+"
+#@overlay/match-child-defaults when=1
+---
+spec:
+  template:
+    spec:
+      containers:
+        #@overlay/match by="name"
+        - name: shutdown-manager
+          resources:
```

Dropping the `when=1` shows that the match doesn't happen since the `resources` block is missing from the upstream contour example yaml. So we need to annotate that if it's missing that's ok

```suggestion
          #@overlay/match missing_ok=True
          resources:
```